add docs for verify-webhook action (#1031)
ngrok/ngrok-docs: traffic-policy/actions/verify-webhook/config.mdx

## Configuration Reference

This is the [Traffic Policy](/docs/http/traffic-policy/) configuration
reference for this action.

### Action Type

`verify-webhook`

### Configuration Fields

| Parameter  | Type     | Description                                                                                                           |
| ---------- | -------- | --------------------------------------------------------------------------------------------------------------------- |

> | `provider` | `string` | **Required.** The name of the provider to verify webhook requests from. Value must be one of the supported providers. |

Reviewer: ```suggestion
| `provider` | `string` | **Required.** The name of the provider to verify webhook requests from. Value must match a supported provider. |
```

> | `secret`   | `string` | A secret key used to validate requests from the given provider.                                                       |

Reviewer: ```suggestion
| `secret`   | `string` | The secret key used to validate webhook requests from the specified provider.                                                   |
```

> | `enforce`  | `bool`   | Whether or not the actions is executed (defaults to true)`.                                                           |

Reviewer: ```suggestion
| `enforce`  | `bool`   | Default `true`. If `false`, continue to the next action even if the request is not allowed.                                                          |
```
